www/webapp/vite.config.js

```diff
@@ -14,6 +15,7 @@ export default defineConfig({
         Components({
             resolvers: [VuetifyResolver()],
         }),
+        legacy(), // Build for old browser.
```

Comment:
How does this interact with `/www/webapp/.browserslistrc` (both the fact that both are there, and the fact that the policies aren't the same)?

Reply:
`vite` uses `esbuild` which does not support `.browserslistrc`.
The legacy build uses broswerslist and creates js for older browsers.

References:
https://vitejs.dev/guide/build.html#browser-compatibility
https://dev.to/meduzen/when-vite-ignores-your-browserslist-configuration-3hoe

Reply:
So we should remove `/www/webapp/.browserslistrc`?

Also, the [default `targets` argument for `legacy`](https://www.npmjs.com/package/@vitejs/plugin-legacy#user-content-targets) is `last 2 versions and not dead, > 0.3%, Firefox ESR`, which is different from what we had in the other file. I'm not saying the change is bad, I'm just making the change explicit.

Reply:
The `target` setting is for the `vite-legacy` package. After generation the chunks are processed again with `babel` which uses `.browserlistrc`.

[Source](
https://github.com/vitejs/vite/tree/06e4f57724f947b584dad68fdd446989bdc76aa0/packages/plugin-legacy#vitejsplugin-legacy-)

> Generate a corresponding legacy chunk for every chunk in the final bundle, transformed with [@babel/preset-env](https://babeljs.io/docs/en/babel-preset-env) and emitted as [SystemJS modules](https://github.com/systemjs/systemjs) (code splitting is still supported!).

The default config of the plugin includes more browser versions than the current config.

So I think the config could be kept, if you wish to have control otherwise it can be removed and trust the default config of the plugin.